### PR TITLE
Fixed seed for randomness, fixed conflicts in errorids

### PR DESCRIPTION
**Pull request by hrkrshnn**
I think the seed for the random number generator was not set correctly. Consider the following situation: Two users creates a branch from develop. Adds a `0000_error` for new errors. If they run `correct_error_ids.py`, they would both get the same error-id as same replacements. Because their seeds were the same. (The new seed is set once, and is a call to system time.)

Also ran `correct_error_ids.py` to fix existing conflicts.

### CODE REVIEW

File: scripts/correct_error_ids.py [hunk 1]
```diff
@@ -117,6 +117,7 @@ def find_source_files(top_dir):
 
 
 def main():
+    random.seed(datetime.now())
```
Comment (a3d4) [inline scripts/correct_error_ids.py]: Could we use just `random.seed()`?

Comment (hrkrshnn) [reply scripts/correct_error_ids.py]: Yes. Didn't know that systime is used as default! I'll fix this.

**Comment by a3d4** [comment]
> I think the seed for the random number generator was not set correctly.

It was intentional. It is often helpful to have reproducible random numbers.

However I wanted to change it for the production system, just the way you did. Thanks a lot!

**Comment by chriseth** [comment]
Are the changed numbers all duplicates?

**Comment by hrkrshnn** [comment]
@chriseth I think so. I did verify a few of them.

**Review by chriseth — APPROVED** [review]

**Comment by a3d4** [comment]
> Are the changed numbers all duplicates?

With the current deterministic script, the same random ids would be generated in two branches. After @hrkrshnn 's fix only incidental collisions are expected.

**Pull request merged**